wallet-new/server/Cardano/Wallet/API/V1/Handlers/Accounts.hs

```diff
@@ -57,23 +57,17 @@ listAccounts RequestParams {..} = do
       }
     _ -> return $ OneOf $ Left example
 
--- | This is an example of how POST requests might look like.
--- It also shows an example of how an error might look like.
--- NOTE: This will probably change drastically as soon as we start using our
--- custom monad as a base of the Handler stack, so the example here is just to
--- give the idea of how it will look like on Swagger.
-newAccount :: WalletId -> Maybe Text -> AccountUpdate -> MonadV1 Account
-newAccount wId _ AccountUpdate{..} = do
-    -- In real code we would generate things like addresses (if needed) or
-    -- any other form of Id/data.
-    newId <- liftIO $ generate arbitrary
-    return Account
-        { accId = newId
-        , accAmount = Core.mkCoin 0
-        , accAddresses = mempty
-        , accName = uaccName
-        , accWalletId = wId
-        }
+newAccount
+    :: (MonadThrow m, V0.MonadWalletLogic ctx m)
+    => WalletId -> Maybe PassPhrase -> AccountUpdate -> m Account
```

Comment:
What about creating a type synonym for `AccountUpdate` and call it `NewAccount` (in `V1.Types` ?) I guess would allow programmers to better understand what's going on.